distribution/adapters/osgi/features/src/main/resources/features.xml

```diff
@@ -79,8 +79,9 @@
     <!-- Keycloak adapter for PaxWeb undertow provider -->
     <feature name="keycloak-pax-http-undertow" version="${project.version}">
         <details>Keycloak Pax-Web adapter for Undertow</details>
-        <feature>keycloak-osgi-adapter</feature>
         <feature>keycloak-adapter-core</feature>
+        <feature version="[7.1,9)">pax-web-core</feature>
```

Comment:
lines 71 and 96 should be fixed in the same manner

Reply:
you mean these commented ones?

Reply:
yes, they're going to be open in community soon